docs/src/main/asciidoc/ansible.adoc

```diff
@@ -0,0 +1,253 @@
+= Automate Quarkus deployment with Ansible
+include::_attributes.adoc[]
+:categories: command-line
+:summary: Build and deploy your Quarkus App using Ansible
+
+
+Let’s see how to build and deploy a Quarkus app using https://docs.ansible.com/ansible/latest/index.html[Ansible]. We’ll see how we can automate the entire process, from the code checkout to the application compilation using Maven and then its deployment and start of the service, as a https://systemd.io/[systemd service], on the target system using Ansible and its collection for Quarkus.
+
+The first part, the application code checkout, compilation and packaging on the Ansible (where Ansible runs). We’ll use the getting-started sample application provided in its {quickstarts-tree-url}/getting-started[Quarkus QuickStarts directory] as a base for this tutorial. We’ll also leverage the https://galaxy.ansible.com/middleware_automation/quarkus[Quarkus collection] for Ansible, an extension for Ansible that alleviates the boilerplate required and to quickly build and deploy a Quarkus using Ansible.
+
+== Prerequisites
+
+:prerequisites-time: 10 minutes
+:prerequisites-no-maven:
+:prerequisites-no-cli:
+include::{includes}/prerequisites.adoc[]
+
+You'll need to https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[install Ansible] on your workstation. Once this is done, you can install this extension for Ansible dedicated to Quarkus with the following command:
+
+[source,bash]
+----
+$ ansible-galaxy collection install middleware_automation.quarkus
+----
+
+=== Inventory file
+
+If you are not familiar with Ansible, please note that the inventory needs to be provided for the command above to run properly. This is a simple text file providing the information Ansible requires on the target system it manages. Please refer to the Ansible documentation for more information on https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html[Ansible inventory].
+
+[source,bash]
+----
+[all]
+
+10.0.0.1
+10.0.0.2
+----
+
+To follow tutorial, you may want to use only one machine (localhost) and skip the ssh authentification setup. This can be easily achieved by using the following inventory file:
```

Comment:
```suggestion
To follow the tutorial, you may want to use only one machine (localhost) and skip the ssh authentication setup. This can be easily achieved by using the following inventory file:
```